README.md

```diff
@@ -25,9 +25,12 @@ Todo JS is a JavaScript project that is intended to showcase an understanding of
 <li>List tasks </li>
 <li>Select tasks  </li>
 <li>drag drop task  </li>
+<li>Add item </li>
+<li>Update Item </li>
+<li>Delete item </li>
 
 ## 🔴 Live Demo <a name = "live"></a>
-**Click [here](https://rawcdn.githack.com/tta2yta/to-do-list/16bf1669a3e9a3d7ba1208cba6f4bad21e96f254/dist/index.html) in order to see a Live Demo of this project.**
+**Click [here](https://rawcdn.githack.com/tta2yta/to-do-list/5427fbc8f0aea8cde594b7daebc92fbe8a008445/dist/index.html) in order to see a Live Demo of this project.**
 
```

Comment:
### [Optional]
It would be nice to add steps to run your app locally and add a screenshot.
Example: 
- git clone `your repo`
- cd into the folder
- git checkout `branch-name`
- npm i 
- npm start
```

Kindly update the README file before merging, 👀  on you.